assets/src/scss/blocks/TableOfContents/TableOfContentsEditorStyle.scss

```diff
@@ -1,20 +1,20 @@
-.submenu-level-link label {
+.table-of-contents-level-link label {
   margin-bottom: 0;
 }
 
-[data-type="planet4-blocks/submenu"] {
+[data-type="planet4-blocks/table-of-contents"] {
```

Comment:
I'm pretty sure this should remain `planet4-blocks/submenu` since we didn't actually change the block name

Reply:
<img width="303" alt="Screenshot 2024-09-12 at 15 44 14" src="https://github.com/user-attachments/assets/901e33be-fa79-4fdc-988c-c10dd4fb9c9c">

Reply:
Well, it says `data-render` because it's in the frontend but you get my point I hope 😬

Reply:
Fixed, thanks!